src/server/db.json

```diff
@@ -8,26 +8,74 @@
     {
       "id": 1,
       "name": "ODM",
-      "value": "OPN"
+      "value": "ODM"
     },
     {
       "id": 2,
       "name": "OBR",
       "value": "OBR"
     }
   ],
-  "skills": [
+  "genericskills": [
```

Comment:
Please use camel case for variable declarations. if it has 2 words. eg; genericSkills